.yarnrc.yml

```diff
@@ -1,3 +1,7 @@
+compressionLevel: mixed
+
+enableGlobalCache: false
```

Comment:
Is the "global cache" referred to here the same cache that we rely upon in CI? If so we might want to leave this enabled.

Reply:
No, this refers to changing yarn to use a global yarn folder for cache, effectively caching dependencies across many projects on your filesystem. 

```
If true, Yarn will disregard the cacheFolder settings and will store the cache files into a folder shared by all local projects sharing the same configuration.

enableGlobalCache: false

```

It also appears to be the default behavior and was just added automatically when upgrading yarn to v4.

Reply:
It seems that the "global cache" is shared between projects, whereas we use a repository-specific cache here (see https://yarnpkg.com/features/offline-cache/#sharing-the-cache). So leaving this disabled makes sense.

Reply:
The logs seem to indicate that the cache is working correctly as well.